lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -61,32 +63,33 @@ def __init__(self, bead_diameter, viscosity=1.002e-3, temperature=20):
         self.viscosity = viscosity
         self.temperature = temperature
 
-    def __call__(self, f, fc, diffusion_constant, f_diode, alpha):
-        return passive_power_spectrum_model(f, fc, diffusion_constant, f_diode, alpha)
-
     def calibration_parameters(self):
         return {
             "Bead diameter": CalibrationParameter("Bead diameter", self.bead_diameter, "um"),
             "Viscosity": CalibrationParameter("Liquid viscosity", self.viscosity, "Pa*s"),
             "Temperature": CalibrationParameter("Liquid temperature", self.temperature, "C"),
         }
 
-    def calibration_results(self, fc, diffusion_constant):
+    def calibration_results(self, fc, diffusion_constant_volts, f_diode, alpha):
         """Compute calibration parameters from cutoff frequency and diffusion constant.
 
         Parameters
         ----------
         fc : float
             Corner frequency, in Hz.
-        diffusion_constant : float
-            Diffusion constant, in (a.u.)^2/s
+        diffusion_constant_volts : float
+            Diffusion constant, in V^2/s
+        f_diode : float
+            Diode frequency.
+        alpha : Alpha
+            Fraction of PSD signal that is instantaneous
```

Comment:
Personally I'm ok with requiring `f_diode` and `alpha` here because models can be called with the same signature regardless of their actual type. Might be worthwhile to add a small note in the docstring though that they're not needed for passive calibration (that it's not an oversight that they're never used)

Reply:
I added a note.